plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationManagerImpl.kt

```diff
@@ -402,10 +431,8 @@ abstract class AnnotationManagerImpl<G : Geometry, T : Annotation<G>, S : Annota
      * false otherwise.
      */
     override fun onMapClick(point: Point): Boolean {
-      if (clickListeners.isEmpty()) {
-        return false
-      }
       queryMapForFeatures(point) {
+        selectAnnotation(it)
```

Comment:
Should we only select annotation if there are interactionListeners in place?

Reply:
We need to update the select state of annotation no matter there are interactionListeners or not.